components/panther.rst

```diff
@@ -0,0 +1,827 @@
+
+Here is a minimal Docker image that can run Panther with both Chrome and
+Firefox:
+
+.. code-block:: dockerfile
+
+    FROM php:alpine
+
+    # Chromium and ChromeDriver
+    ENV PANTHER_NO_SANDBOX 1
+    # Not mandatory, but recommended
+    ENV PANTHER_CHROME_ARGUMENTS='--disable-dev-shm-usage'
+    RUN apk add --no-cache chromium chromium-chromedriver
+
+    # Firefox and GeckoDriver (optional)
+    ARG GECKODRIVER_VERSION=0.28.0
+    RUN apk add --no-cache firefox libzip-dev; \
+        docker-php-ext-install zip
+    RUN wget -q https://github.com/mozilla/geckodriver/releases/download/v$GECKODRIVER_VERSION/geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz; \
+        tar -zxf geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz -C /usr/bin; \
+        rm geckodriver-v$GECKODRIVER_VERSION-linux64.tar.gz
+
+You can then build and run your image:
+
+.. code-block:: bash
+
+    $ docker build . -t myproject
+    $ docker run -it -v "$PWD":/srv/myproject -w /srv/myproject myproject bin/phpunit
+
+Integrating Panther In Your CI
+------------------------------
+
+Github Actions
+~~~~~~~~~~~~~~
+
+Panther works out of the box with `GitHub Actions`_.
+Here is a minimal ``.github/workflows/panther.yaml`` file to run Panther tests:
+
+.. code-block:: yaml
+
+    name: Run Panther tests
+
+    on: [ push, pull_request ]
+
+    jobs:
+      tests:
+
+        runs-on: ubuntu-latest
+
+        steps:
+          - uses: actions/checkout@v2
+
+          - name: Install dependencies
+            run: composer install -q --no-ansi --no-interaction --no-scripts --no-progress --prefer-dist
+
+          - name: Run test suite
+            run: bin/phpunit
+
+Travis CI
+~~~~~~~~~
+
+Panther will work out of the box with `Travis CI`_ if you add the Chrome addon.
+Here is a minimal ``.travis.yaml`` file to run Panther tests:
+
+.. code-block:: yaml
+
+    language: php
+    addons:
+      # If you don't use Chrome, or Firefox, remove the corresponding line
+      chrome: stable
+      firefox: latest
+
+    php:
+      - 8.0
+
+    script:
+      - bin/phpunit
+
+Gitlab CI
+~~~~~~~~~
+
+Here is a minimal ``.gitlab-ci.yaml`` file to run Panther tests
+with `Gitlab CI`_:
+
+.. code-block:: yaml
+
+    image: ubuntu
+
+    before_script:
+      - apt-get update
+      - apt-get install software-properties-common -y
+      - ln -sf /usr/share/zoneinfo/Europe/Paris /etc/localtime
+      - apt-get install curl wget php php-cli php7.4 php7.4-common php7.4-curl php7.4-intl php7.4-xml php7.4-opcache php7.4-mbstring php7.4-zip libfontconfig1 fontconfig libxrender-dev libfreetype6 libxrender1 zlib1g-dev xvfb chromium-chromedriver firefox-geckodriver -y -qq
+      - export PANTHER_NO_SANDBOX=1
+      - export PANTHER_WEB_SERVER_PORT=9080
+      - php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+      - php composer-setup.php --install-dir=/usr/local/bin --filename=composer
+      - php -r "unlink('composer-setup.php');"
+      - composer install
```

Comment:
shall we use the same options like used in the panther.yaml GithubActions file?

Reply:
Updated everywhere